Clojure Tests Are Breaking

I thought maybe this was a short term issue, but I inspected DockerHub. It looks like they uploaded new images 3 hours ago, and now our tests are breaking. 🔥

It looks like they may have changed the command needed to execute these scripts, or this is is just a bug:

```
E         - WARNING: When invoking clojure.main, use -M
```

--- a/archive/c/clojure/testinfo.yml
+++ b/archive/c/clojure/testinfo.yml
@@ -4,5 +4,5 @@ folder:
 
 container:
   image: "clojure"
-  tag: "tools-deps"
+  tag: "openjdk-16-tools-deps-alpine"
   cmd: "clojure ./{{ source.name}}{{ source.extension }}"

--- a/archive/c/clojure/testinfo.yml
+++ b/archive/c/clojure/testinfo.yml
@@ -4,5 +4,5 @@ folder:
 
 container:
   image: "clojure"
-  tag: "openjdk-16-tools-deps-alpine"
+  tag: "tools-deps-1.10.1.561"
   cmd: "clojure ./{{ source.name}}{{ source.extension }}"

